### ISSUE TITLE
RuntimeError in README example

### ISSUE THREAD

**Issue by prabathbr** [francois-rozet/piqa#35]
### Description

I have tried to run given SSIM example in README but it gives below error. 

### Reproduce

A minimal working example demonstrating the current behavior.

```python
import torch
from piqa.ssim import ssim
from piqa.utils.functional import gaussian_kernel

x = torch.rand(5, 3, 256, 256, requires_grad=True).cuda()
y = torch.rand(5, 3, 256, 256).cuda()

kernel = gaussian_kernel(11, sigma=1.5).expand(3, 11, 11)

l = 1 - ssim(x, y, kernel=kernel)
```

### Expected behavior

Correct output for `l` with SSIM loss, which can be obtained with below code.

```
import torch
import piqa

# SSIM
x = torch.rand(5, 3, 256, 256, requires_grad=True).cuda()
y = torch.rand(5, 3, 256, 256).cuda()

ssim = piqa.SSIM(window_size=11, sigma=1.5, n_channels = 3, reduction = 'mean').cuda()
l = 1 - ssim(x, y)
l.backward()

print(l)
```

output : `tensor(0.9923, device='cuda:0', grad_fn=<RsubBackward1>)`

### Possible issue

Error in the example code, possibly with number of channels and Gaussian kernel mapping.

### Environment

* PIQA version: 1.3.1
* PyTorch version: 2.1.0+cu118
* Python version: 3.9.18
* OS:  Windows10

### Error

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[1], line 10
      6 y = torch.rand(5, 3, 256, 256).cuda()
      8 kernel = gaussian_kernel(11, sigma=1.5).expand(3, 11, 11)
---> 10 l = 1 - ssim(x, y, kernel=kernel)

File k:\Miniconda3\envs\pytorch\lib\site-packages\torch\jit\_trace.py:1230, in _script_if_tracing.<locals>.wrapper(*args, **kwargs)
   1226 @functools.wraps(fn)
   1227 def wrapper(*args, **kwargs):
   1228     if not is_tracing():
   1229         # Not tracing, don't do anything
-> 1230         return fn(*args, **kwargs)
   1232     compiled_fn = script(wrapper.__original_fn)  # type: ignore[attr-defined]
   1233     return compiled_fn(*args, **kwargs)

File k:\Miniconda3\envs\pytorch\lib\site-packages\piqa\ssim.py:102, in ssim(x, y, kernel, channel_avg, padding, value_range, k1, k2)
     99     pad = 0
    101 # Mean (mu)
--> 102 mu_x = channel_convs(x, window, pad)
    103 mu_y = channel_convs(y, window, pad)
    105 mu_xx = mu_x ** 2

File k:\Miniconda3\envs\pytorch\lib\site-packages\piqa\utils\functional.py:86, in channel_convs(x, kernels, padding)
     83     x = F.pad(x, pad=pad)
     85 for k in kernels:
---> 86     x = channel_conv(x, k)
     88 return x

File k:\Miniconda3\envs\pytorch\lib\site-packages\piqa\utils\functional.py:46, in channel_conv(x, kernel, padding)
     44     return F.conv3d(x, kernel, padding=padding, groups=x.shape[-4])
     45 elif D == 2:
---> 46     return F.conv2d(x, kernel, padding=padding, groups=x.shape[-3])
     47 elif D == 1:
     48     return F.conv1d(x, kernel, padding=padding, groups=x.shape[-2])

RuntimeError: Given groups=3, weight of size [3, 11, 11, 1], expected input[5, 3, 256, 256] to have 33 channels, but got 3 channels instead
```

**Comment by francois-rozet** [comment]
Hello @prabathbr :wave: Thank you for the bug report. Indeed it should be

```python
kernel = gaussian_kernel(11, sigma=1.5).repeat(3, 1, 1)
```

This example has been there for years and no one noticed the mistake :facepalm:

**Comment by francois-rozet** [comment]
Fixed in the latest version (d512c86f4af845c4f54f86a2f0ef8866851b8f5c)
